ci/pipelines/apps-test-ecr.yml

```diff
@@ -316,30 +345,34 @@ jobs:
             args:
               - -ec
               - |
-                echo "Imagine we just smoked on test."
-      - task: combine-passing-tags
+                echo "Imagine we just deployed to test."
+  - name: smoke-test-toolbox
+    plan:
+      - get: toolbox-ecr-registry-test
+        trigger: true
+        passed: [deploy-toolbox]
+      - task: smoke-test-on-test
         config:
           platform: linux
           image_resource:
             type: registry-image
             source:
               repository: govukpay/concourse-runner
-          inputs:
-            - name: toolbox-git-release
           run:
             path: /bin/bash
             args:
               - -ec
               - |
-                COMMIT_SHA=$(cat toolbox-git-release/.git/HEAD)
-                GIT_TAG=$(cat toolbox-git-release/.git/ref)
-                echo "$COMMIT_SHA $GIT_TAG ready-staging" > docker_tags/docker_tags.txt
-          outputs:
-            - name: docker_tags
-      - put: toolbox-ecr-registry-test
+                echo "Imagine we just smoked on test."
+  - name: push-toolbox-to-staging-ecr
+    plan:
+      - get: toolbox-ecr-registry-test
+        trigger: true
+        passed: [smoke-test-toolbox]
+      - put: toolbox-ecr-registry-staging
         params:
           image: image/image.tar
-          additional_tags: docker_tags/docker_tags.txt
+
   - name: frontend-image-to-test-ecr
     plan:
       - get: apps-test-ecr
```

Comment:
Does each job need to `get: apps-test-ecr`, isn't this the git resource watching the config of this pipeline itself?

![Screenshot 2021-02-17 at 16 02 31](https://user-images.githubusercontent.com/31340341/108231368-99107800-7139-11eb-80ce-de441d2c1cc0.png)

Reply:
sorry, a) this PR is for toolbox b) its used in `*pull-image-from-dockerhub`...